xbmc/pvr/PVRGUIInfo.cpp

```diff
@@ -686,11 +707,23 @@ void CPVRGUIInfo::CharInfoPlayingClientName(std::string &strValue) const
 
 void CPVRGUIInfo::CharInfoEncryption(std::string &strValue) const
 {
-  CPVRChannelPtr channel(CServiceBroker::GetPVRManager().Clients()->GetPlayingChannel());
-  if (channel)
-    strValue = channel->EncryptionName();
+  if (m_descrambleInfo.iCaid != -1)
```

Comment:
thought about it, but then we also need constants for invalid provid, hops, ecmtime, ... do we want this?